src/Avalonia.DesignerSupport/Remote/PreviewerWindowingPlatform.cs

```diff
@@ -21,6 +21,10 @@ class PreviewerWindowingPlatform : IWindowingPlatform
         public ITrayIconImpl CreateTrayIcon() => null;
 
         public IWindowImpl CreateWindow() => new WindowStub();
+        public ITopLevelImpl CreateEmbeddableTopLevel()
+        {
+            throw new NotImplementedException();
```

Comment:
```suggestion
            return CreateEmbeddableWindow();
```

will work here and possible on other platforms too